src/main/java/io/spring/githubchangeloggenerator/github/service/GitHubService.java

```diff
@@ -45,7 +45,7 @@
 
 	private static final Pattern LINK_PATTERN = Pattern.compile("<(.+)>; rel=\"(.+)\"");
 
-	private static final String MILESTONES_URI = "/repos/{owner}/{name}/milestones";
+	private static final String MILESTONES_URI = "/repos/{owner}/{name}/milestones?state=all&sort=due_on&direction=desc&per_page=50";
```

Comment:
I am a bit nervous that `sort=due_on` would show milestones that don't have a due date first as it is the case on the Web UI. For projects with a large number of milestones, this may be a problem.